### PR TITLE
chore: lint code & non-code files

### PR DESCRIPTION
**Pull request by JaeAeich**
lint code and non code files

### CODE REVIEW

File: .github/workflows/build_and_publish_charts.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (uniqueg) [inline .github/workflows/build_and_publish_charts.yaml:1]: Not sure why you add the YAML open, but not the closing tags?

File: deployment/charts/tesk/s3-config/credentials-TEMPLATE [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [default]
```
Comment (uniqueg) [inline deployment/charts/tesk/s3-config/credentials-TEMPLATE:1]: I guess/hope this is not a real secret? Might be good to clearly indicate that by having it start with `DUMMYKEY` or something

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (uniqueg) [inline .pre-commit-config.yaml:1]: How about including a check for the sort order of `import` statements?

**Review by uniqueg — COMMENTED** [review]
Nice! 

However, I would rename the PR to something like "chore: lint code & non-code files", given that the vast majority of changes are more directly related to linting, and the addition of the pre-commit config is just a way to enforce this. 

Alternatively, create two PRs out of this: In the first you do the various linting changes to the files, in the second you only add the pre-commit config.

File: .pre-commit-config.yaml (hunk 3 above)
Comment (JaeAeich) [reply .pre-commit-config.yaml:1]: ruff, `Isort` is already doing that :)

File: deployment/charts/tesk/s3-config/credentials-TEMPLATE (hunk 2 above)
Comment (JaeAeich) [reply deployment/charts/tesk/s3-config/credentials-TEMPLATE:1]: :+1:

**Review by uniqueg — APPROVED** [review]
Looks good to me, but please resolve the conflicts.

**Review by uniqueg — APPROVED** [review]

**Pull request merged**